src/main/java/pl/lukakan/contactform/MailService.java

```diff
@@ -0,0 +1,31 @@
+package pl.lukakan.contactform;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.mail.javamail.MimeMessageHelper;
+import org.springframework.stereotype.Service;
+
+import javax.mail.MessagingException;
+import javax.mail.internet.MimeMessage;
+
+@Service
+public class MailService {
+    private final JavaMailSender javaMailSender;
+
+    @Autowired
+    public MailService(JavaMailSender javaMailSender) {
+        this.javaMailSender = javaMailSender;
+    }
+
+    public void sendMail(Message message, String targetEmailAddress, String senderEmailAddress) throws MessagingException {
```

Comment:
Wydaje mi się, że trochę za bardzo chciałeś reużyć pewne klasy/metody, w efekcie czego zrobił się lekki bałagan, moja propozycja jest taka:
1. Klasa `Message` została wykorzystana na 2 sposoby: jako model formularza (na adresie "/contact") oraz coś co jest przekazywane tuta. Efektem tego jest to, ze istnieje pole `body` potrzebne tylko tu i pole `content`, potrzebne tylko w formularzu. Stwórz klasę, np. `SendMessageForm`, która będzie wykorzystywana tylko jako model formularza i na pewno nie będzie przekazywana tutaj ;)
2. Tutaj wydaje mi się że potrzebujesz 4 rzeczy: tytułu, nadawcy (jest to jednoczesnie replyTo), odbiorcy oraz samj treści. Przyjmuj tutaj tylko te 4 informacje (mogą one być spakowane w jednej klasie).